EIPS/eip-7612.md

```diff
@@ -0,0 +1,181 @@
+eip: 7612
+title: State storage transition using an overlay tree
+description: Describes the use of an overlay tree to use the verkle tree structure, while leaving the historical state untouched.
+author: Guillaume Ballet (@gballet), Ansgar Dietrichs (@), Ignacio Hagopian (@jsign), Gottfried Herold (@GottfriedHerold), Jamie (@), Tanishq Jasoria (@tanishqjasoria), Parithosh Jayanthi (@parithosh), Gabriel Rocheleau (@gabrocheleau), Karim Taam (@matkt)
+discussions-to: <URL>
```

Comment:
Need to create a discussions topic in https://ethereum-magicians.org/ and then add the URL here.  
Topic can be as simple as a link to this PR.